qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml

```diff
@@ -7,14 +7,5 @@ tasks:
     client.0:
       rgw_server: client.0
       storage classes: LUKEWARM, FROZEN
-      extra_attrs: ["lifecycle"]
       lc_debug_interval: 10
-overrides:
-  ceph:
-    conf:
-      client:
-        debug rgw: 20
-        rgw lc debug interval: 10
-        storage classes: LUKEWARM, FROZEN
-  rgw:
-    storage classes: LUKEWARM, FROZEN
```

Comment:
Isn't this needed to configure storage classes on rgw server?

Reply:
these overrides are already present in qa/suites/rgw/lifecycle/overrides.yaml, so they're not needed here